src/main/java/com/webonise/models/Coordinates.java

```diff
@@ -0,0 +1,50 @@
+package com.webonise.models;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Observable;
+
+@Component
+public class Coordinates extends Observable {
+    private float latitude;
+    private float longitude;
```

Comment:
Avoid using float.
Floats are not equal safe.
http://stackoverflow.com/questions/1088216/whats-wrong-with-using-to-compare-floats-in-java